### ISSUE TITLE
Update Cordova docs for minimum supported Xcode version

### ISSUE THREAD

**Issue by ajaypillay** [apache/cordova#230]
This [page](https://cordova.apache.org/docs/en/latest/guide/platforms/ios/index.html#requirements-and-support) states that the minimum Xcode version is 8.0 which runs on OS X El Capitan.

However, I am building an application in Meteor with Cordova bundled in and I get an error upon compiling stating:
`✗ Xcode: Cordova needs xcodebuild version 9.0.0 or greater, you have version
  8.2. Please install version 9.0.0 or greater from App Store
`
I am running mac OS 10.11.6. Thanks.

**Comment by breautek** [comment]
I agree the page should be updated, PR is welcome. The minimum required xcode is now Xcode 11, which is an apple requirement, not necessary a Cordova requirement.

Also note that Apple doesn't support El Capitan, you'll need to update your Mac OS to gain access to the latest xcode. If you don't have Mac OS update available, it's because you're using a mac model deemed vintage by Apple and they've been cut off from updates. Their last last cutoff was 2012-2015 devices (depending on whether they are iMacs, or MacBooks, etc). This essentially means that you need to buy a newer mac computer.

[See supported Mac hardware for Catalina](https://support.apple.com/en-ca/HT210222)
